### PR TITLE
Feat; add ConfirmationModal and GridBox

### PR DESCRIPTION
**Pull request by sbracegirdle**

### CODE REVIEW

File: src/components/ConfirmationModal/ConfirmationModal.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            noMargin
+            color="core.content.contentSecondary"
+          >
+            {text}
+          </Typography>
+        </Box>
+      </Box>
+    </StyledModalContent>
+    <StyledModalFeet>
+      <Box
```
Comment (symonomicon) [inline src/components/ConfirmationModal/ConfirmationModal.tsx:108]: q; would it support a button group something like this?  
https://www.figma.com/file/pQqcx1z6rSV8MVmokydQMr/%F0%9F%93%90-Bipartisan-Design-System?node-id=812-4280&t=oJ692gMr22MTLCEa-4

where one btn is way left and the rest are on right, or maybe vise versa

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@govconnex/ui",
-  "version": "0.0.102",
+  "version": "0.0.105",
```
Comment (symonomicon) [inline package.json:3]: Q; is it meant to jump 3 versions 👀

Comment (sbracegirdle) [reply package.json:3]: Yes

File: src/components/ConfirmationModal/ConfirmationModal.tsx (hunk 1 above)
Comment (sbracegirdle) [reply src/components/ConfirmationModal/ConfirmationModal.tsx:108]: Yes, eventually. Out of scope for this PR though, this is just the base functionality.

**Review by symonomicon — APPROVED** [review]

**Pull request merged**